### DISCUSSION TITLE
是否能支持默认的Vivado生成的结构，或者增加选择工程结构的选项

### DISCUSSION THREAD

**Discussion by alwaysmy** [Digital-EDA/Digital-IDE#59] [Ideas]
有的时候有的已经建好的vivado工程不想迁移或者是有时候合作的时候用的是vivado的默认的工程结构，似乎这样可以增加一个选项来选择当前是直接在已有的vivado工程下编辑还是按照DigitalIDE工程生成新的文件目录结构来生成（也许还有其他工具的工程文件目录结构）？

b.t.w. 也许是本来就支持了这种工程不过我没找到在哪里？这里似乎有个bug存在影响了我的判断：似乎是在vivado默认工程结构下的`projectname.src/sim_1/new/xxx.v`这样的文件仿真的时候会提示 is not a hdfFile? 是否是因为匹配到了路径上的`.src`上

**Comment by Nitcloud** [comment]
为了能兼容IC设计和后续其他国产FPGA设计，我们将不准备专门为vivado构建一个专门的文件结构，我们后续会强推我们的标准文件结构，这将在统一IC&FPGA设计是很有意义的，对于此，我们将会设计自动化的工具来帮助你将vivado的文件结构转换为标准文件结构。

**Reply by Nitcloud** [reply]
这在0.4.0中会实现，请期待。
